source/getting-started-edge/index.rst

```diff
@@ -0,0 +1,245 @@
+via federated login <federation>`).
+
+On your first Chameleon login you will be asked to accept `terms and conditions
+<https://auth.chameleoncloud.org/auth/realms/chameleon/terms>`_ of use. Please,
+note that as part of those terms and conditions you are requested to acknowledge
+Chameleon in publications produced using the testbed: see our FAQ for
+information on `how to reference Chameleon in your publications
+<https://www.chameleoncloud.org/learn/frequently-asked-questions/#toc-how-should-i-reference-chameleon->`_
+and the suggested `acknowledgement text
+<https://www.chameleoncloud.org/learn/frequently-asked-questions/#toc-how-should-i-acknowledge-chameleon-in-my-publications->`_.
+
+Once you log in, you will be able to :ref:`edit your Chameleon profile
+<profile-page>`, sign up for webinars, and participate in our community.
+However, to actually use the testbed you will first need to **join or create a
+project** (see below).
+
+.. _getting-started-project:
+
+Step 2: Create or join a project
+================================
+
+To get access to Chameleon resources, you will need to be associated with a
+**project** that is assigned a **resource allocation**.
+
+If you want to **join an existing Chameleon project**, you will need to ask the PI
+of the project to add your username. You can find your username in `your Chameleon profile
+<https://www.chameleoncloud.org/user/profile/>`_--it is also displayed in the
+top-right corner when you are logged in.
+
+If you want to **create a project**, you will first either need to obtain a PI
+status or work with somebody who has PI status. To determine if you can obtain
+PI status, please see a :ref:`list of PI eligibility criteria <pi-eligibility>`.
+If you do not meet these criteria (e.g., students generally do not), you will
+need to ask your advisor or other scientist supervising your research to create
+the project for you. You can request PI status by checking a box in `your
+Chameleon profile <https://www.chameleoncloud.org/user/profile/>`_. Chameleon PI
+status requests are typically reviewed within one business day.
+
+Once you have PI status, you may apply for a new project with an initial
+allocation. A project application typically consists of a short description of
+your intended research and takes one business day to process. Once your project
+has been approved, you will be able to utilize the testbed sites.
+
+Step 3: Start using Chameleon!
+==============================
+
+Congratulations, you are now ready to launch your first container! Containers
+are a simple way to deploy applications. Learn more about what containers are 
+with `this guide from docker <https://www.docker.com/resources/what-container>`_.
+Follow these steps to launch a container and manage it.
+
+The CHI\@EDGE dashboard
+----------------------
+
+Chameleon edge resources are available through 
+`CHI@EDGE <https://chi.edge.chameleoncloud.org>`_. When you access this site, you are 
```

Comment:
```suggestion
`CHI@Edge <https://chi.edge.chameleoncloud.org>`_. When you access this site, you are 
```

We'll probably use "Edge" as it is not an acronym. But it would be funny to imagine an acronym for EDGE. "Edge Device General Environment"? Hehe